src/App.tsx

```diff
@@ -63,6 +84,37 @@ const checkoutTheme = createTheme({
         },
       },
     },
+    MuiOutlinedInput: {
+      styleOverrides: {
+        root: {
+          borderRadius: "8px",
+        },
+      },
+    },
+    MuiLoadingButton: {
+      styleOverrides: {
+        loadingIndicator: {
+          color: "white !important",
+        },
+        root: ({ ownerState }) =>
+          (ownerState.loading && {
+            backgroundColor: "#0B3EE3 !important",
```

Comment:
To avoid repeating the hex color string, you can use the theme argument provided in the root callback.

Reply:
Thanks for the useful info! I was trying to understand how to do it.